galpy/potential/TwoPowerSphericalPotential.py

```diff
@@ -8,11 +8,14 @@
 ###############################################################################
 import math as m
 import numpy
-from scipy import special, optimize, integrate
+from numpy import sqrt, pi
+from scipy import optimize, integrate
+from scipy.special import gamma as Gamma, hyp2f1
```

Comment:
To remember where these functions come from, just import ``special`` and use these functions as ``special.``

Reply:
done.